init.sh

```diff
@@ -0,0 +1,10 @@
+if [! -d .oh-my-zsh]
+then 
+	sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
+fi
+ln -s .dotfiles/.vimrc $HOME/.vimrc
+ln -s .dotfiles/.zshrc $HOME/.zshrc
+rm $ZSH_CUSTOM/aliases.zsh
```

Comment:
Are you referring to .oh-my-zsh? What can I do to handle this case more gracefully? Specify the location for it to be installed?